apps/osim/workflows/rejected.yml

```diff
@@ -0,0 +1,14 @@
+---
+name: REJECTED
+description: >
+  vulnerability does not affect any Red Hat or community products
+  the flaw was analysed but all affects were resolved as notaffected
+  there is no follow-up action required after resolving the affects
```

Comment:
nitpick: I think that this block of text will compile into a single paragraph connecting the lines by spaces when read into memory and then it would make a weird sentence when the sub-sentences are not joint in human readable format (comma or conjunction). However I am not sure it is a real issue so feel free to ignore this comment.

Reply:
I will add some pauses (commas / breaks) in the text, those descriptions are not being used anywhere in the code right now, but I believe it will be better even as just "code comment", thank you.